internal/configs/sharding/mainnet.go

```diff
@@ -28,6 +28,7 @@ const (
 	mainnetV1_3Epoch = 36
 	mainnetV1_4Epoch = 46
 	mainnetV1_5Epoch = 54
+	mainnetV2Epoch   = 177
```

Comment:
Actually, this should be 178. Should be the same as staking epoch. Not pre-staking epoch.

Reply:
But since you had the same config (the voting power doesn't take effect yet). It shouldn't be the cause of the upgrade issue.